### DISCUSSION TITLE
UX - adding table columns

### DISCUSSION THREAD

**Discussion by jozef-slezak** [dineug/erd-editor#277] [Ideas]
I propose following small improvements for better user experience while adding new columns:
- [ ] autofocus on inputs (at least in name of the column) after clicking add column (to avoid double clicking)
- [ ] add button near delete button (to simplify mouse movement)
- [ ] keyboard shortcut for adding new column

![image](https://user-images.githubusercontent.com/16844103/133734252-d5a03634-710f-4f98-bad4-b4db13c7cf91.png)

**Comment by dineug** [comment]
There is a default supported keymap.

add column - alt + enter
edit - enter
remove column - alt + [delete or backspace]
focus move - arrow key
edit focus move - tab

**Comment by jozef-slezak** [comment]
Thanks for the quick reply. I will use keyboard shortcuts

Our business analyst would prefer less clicks therefore I have created this
issue. I believe many of the users would.

**Comment by dineug** [comment]
I understood.

However, we design the UI/UX with the following principles.

We do not create duplicate functional UI.
It should be as simple as possible and does not expose unnecessary information.

I can consider autofocus on inputs,
I think it's unnecessary because it already supports enough keymaps.
